installers/LinuxInstaller/launcher-template.desktop.in

```diff
@@ -1,9 +1,3 @@
-// Mantid Repository : https://github.com/mantidproject/mantid
```

Comment:
Whoops! I'm guessing this happened to due the [copyrightupdate.py](https://github.com/mantidproject/mantid/blob/master/tools/Copyright/copyrightupdate.py#L46) script when we added the CSNS. Could update the excluded files list in the linked script so that this doesn't happen again?